examples/bookstore.proto.output.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "google/api/annotations.proto";
+
+option go_package = "/newbookstore";
+
+message book {
```

Comment:
```suggestion
message Book {
```

Reply:
ah, thanks! this is generated via the go code, so I can fix that. I'll add a validator too so we can catch this next time (I think the resource `kind` should be capitalized.

Reply:
look at the yaml output for now: proto doesn't have all the annotations we need yet.